### ISSUE TITLE
(cais): Remove calls to tf.constant() once the gradients functions 

### ISSUE THREAD

**Issue by piascikj** [imdone/tensorflow#4183]
accept lists and np.ndarrays.

----
*opened via [imdone.io](https://imdone.io) from a code comment on [65866f04c711ecb5e14df176f1c6ebbf51ebafc4](https://github.com/imdone/tensorflow/commit/65866f04c711ecb5e14df176f1c6ebbf51ebafc4) by Shanqing Cai cais@google.com*

----

https://github.com/imdone/tensorflow/blob/8cb6e535f6ea14b380aa86d425169adad682cb8c/tensorflow/python/eager/backprop.py#L212-L218
